src/hooks/chat.ts

```diff
@@ -32,7 +32,7 @@ export default (
       return useQuery({
         queryKey: buildItemChatKey(itemId),
         queryFn: () =>
-          Api.getItemChat(itemId, queryConfig).then((data) => Map(data)),
+          Api.getItemChat(itemId, queryConfig).then((data) => Map<unknown, string|ChatMessage[]>(data)),
```

Comment:
Do we still need that?

Reply:
I removed it in 68b7524 and the Chatbox did not complain. I think we can remove it.